src/archanaNJun20/PhoneNumberValidation.java

```diff
@@ -0,0 +1,95 @@
+				}
+				if(flag==false)		
+					System.out.println(phoneNumber +" InVaild number");
+				else	
+					System.out.println(phoneNumber +" Vaild number");
+			}else
+				System.out.println(phoneNumber +" InVaild number");
+			break;
+		case 12:
+
+			if(phoneNumber.startsWith("9")||phoneNumber.startsWith("8")||phoneNumber.startsWith("7") && phoneNumber.charAt(3)==' ' && phoneNumber.charAt(7)==' '){
+				for(int index=1;index<length;index++) {
+					if(Character.isLetter(phoneNumber.charAt(index))==true)
+						flag=false;
+				}
+				if(flag==false)		
+					System.out.println(phoneNumber +" InVaild number");
+				else	
+					System.out.println(phoneNumber +" Vaild number");
+			}else
+				System.out.println(phoneNumber +" InVaild number");
+			break;
+		case 14:
+			if(phoneNumber.startsWith("+91 ") && phoneNumber.charAt(4)=='9'|| phoneNumber.charAt(4)=='8'||phoneNumber.charAt(4)=='7' ){
+				for(int index=4;index<length;index++) {
+					if(Character.isLetter(phoneNumber.charAt(index))==true || phoneNumber.charAt(index)==' '){	
+						flag=false;
+						break;
+					}
+				}
+				if(flag==false)		
+					System.out.println(phoneNumber +" InVaild number");
+				else	
+					System.out.println(phoneNumber +" Valid number");
+			}else
+				System.out.println(phoneNumber +" InVaild number");
+			break;
+
+		default  :System.out.println(phoneNumber +" Invalid phone number");
+		break;
+		}
+	}
+	public static void main(String[] args) {
+		PhoneNumberValidation phoneNumberValidation=new PhoneNumberValidation();
+		System.out.println("phone number of length :10");
+		phoneNumberValidation.checkPhoneNumberValidation("9028553461");
+		phoneNumberValidation.checkPhoneNumberValidation("90 8553461");
+		phoneNumberValidation.checkPhoneNumberValidation("90y8553461");
+		phoneNumberValidation.checkPhoneNumberValidation("2028553461");
+		System.out.println("\n" +"phone number of length :11");
+		phoneNumberValidation.checkPhoneNumberValidation("09028553461");
+		phoneNumberValidation.checkPhoneNumberValidation("090 8553461");
+		phoneNumberValidation.checkPhoneNumberValidation("90y8553461");
+		phoneNumberValidation.checkPhoneNumberValidation("1028553461");
+		phoneNumberValidation.checkPhoneNumberValidation("02028553461");
+		System.out.println("\n" +"phone number of length :12");
+		phoneNumberValidation.checkPhoneNumberValidation("902 855 3461");
+		phoneNumberValidation.checkPhoneNumberValidation("9028 55 3461");
+		phoneNumberValidation.checkPhoneNumberValidation("90i 855 3461");
+		phoneNumberValidation.checkPhoneNumberValidation("202 855 3461");
+		System.out.println("\n" +"phone number of length :14");
+		phoneNumberValidation.checkPhoneNumberValidation("+91 9028553461");
+		phoneNumberValidation.checkPhoneNumberValidation("+91 90 8553461");
+		phoneNumberValidation.checkPhoneNumberValidation("+91 90y8553461");
+		phoneNumberValidation.checkPhoneNumberValidation("+91 9028553461");
+		phoneNumberValidation.checkPhoneNumberValidation("+91 2028553461");
```

Comment:
Instead of calling same method multiple times you can create an array of String and call the method in the loop and there is no need to print the length of the Mobile Number.

Reply:
Hi Aavruti, The code has updated as per your given comment.